lib/fluent/plugin/in_wmi.rb

```diff
@@ -7,25 +7,34 @@
 module Fluent::Plugin
   class WmiInput < Fluent::Plugin::Input
     Fluent::Plugin.register_input("wmi", self)
+    helpers :timer
 
     desc "The value is the tag assigned to the generated events."
     config_param :tag, :string
-
     desc "The value is the instance name of WMI."
     config_param :class_name, :string
+    desc "Determine the rate to emit metrics as events."
+    config_param :interval, :time, default: 10
 
     def configure(conf)
       super
-
       raise Fluent::ConfigError, "class_name is required." unless @class_name
```

Comment:
Without :default hash key, config_param always checks whether the value is provided or not.
So, this validator is needless.

Reply:
My next plan would be to add "query" and specify either "class_name" or "query". Therefore, the logic of validation here is expected to change.